Doc/whatsnew/3.11.rst

```diff
@@ -825,6 +825,18 @@ time
   it had a resolution of 1 millisecond (10\ :sup:`-3` seconds).
   (Contributed by Benjamin Szőke, Dong-hee Na, Eryk Sun and Victor Stinner in :issue:`21302` and :issue:`45429`.)
 
+traceback
+---------
+
+* Add :func:`traceback.StackSummary.format_frame_summary` to allow users
+  to override which frames appear in the traceback, and how they are
+  formatted.
+  (Contributed by Ammar Askar in :issue:`44569`.)
+
+* Add :func:`traceback.TracebackException.print` which prints the
+  formatted ``TracebackException`` instance to a file.
```

Comment:
It might be helpful to make this an actual reference, since the exception class isn't already referenced here, just one of its methods:

```suggestion
  formatted :exc:`~traceback.TracebackException` instance to a file.
```

Otherwise, if we don't want the link, we should at least be stylistically consistant with the formatting and semantically consistent with the markup:

```suggestion
  formatted :exc:`~!traceback.TracebackException` instance to a file.
```

Reply:
(See python/docs-community#52 for related discussion)